iroh-relay/src/relay_map.rs

```diff
@@ -131,6 +161,28 @@ pub struct RelayNode {
     pub quic: Option<RelayQuicConfig>,
 }
 
+impl From<RelayUrl> for RelayNode {
+    fn from(value: RelayUrl) -> Self {
+        Self {
+            url: value,
+            stun_only: false,
+            stun_port: DEFAULT_STUN_PORT,
+            quic: quic_config(),
+        }
+    }
+}
+
+impl From<(RelayUrl, u16)> for RelayNode {
```

Comment:
Again, I'm only so-so convinced this is worth it rather than more confusing.  It made more sense when there only was STUN before, but now with QAD?

Reply:
agreed on all your points about the `(RelayUrl, u16)` impls, I'm removing them. It looks like the original code was only used in `iroh-doctor` and can easily be adjusted.